hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html

```diff
@@ -48,6 +48,8 @@ <h2>Node Status</h2>
         <tr>
             <th ng-click = "columnSort('hostname')" class="nodeStatusInfo"><span ng-class = "{'sorting' : (columnName != 'hostname'), 'sortasc' : (columnName == 'hostname' && !reverse),
                                         'sortdesc':(columnName == 'hostname' && !reverse)}">HostName</span></th>
+            <th ng-click = "columnSort('usedpercent')" class="nodeStatusInfo" ><span ng-class="{'sorting' : (columnName != 'usedpercent'), 'sortasc' : (columnName == 'usedpercent' && !reverse),
+                                        'sortdesc':(columnName == 'usedpercent' && !reverse)}">Used Percent</span></th>
```

Comment:
Nit: Wouldn't it be more intuitive if we renamed it to `Used Space Percent` or `Used Space Percentage`? Or is it too long and it breaks the UI?